auth/util/src/main/java/org/wildfly/security/auth/util/CaseNameRewriter.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2020 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wildfly.security.auth.util;
+
+import java.util.Locale;
+import org.wildfly.security.auth.server.NameRewriter;
+
+/**
+ * A case name rewriter adjusts a principal to upper or lower case.
+ *
+ * @author <a href="mailto:szaldana@redhat.com">Sonia Zaldana Calles</a>
+ */
+public final class CaseNameRewriter implements NameRewriter {
+    private final boolean upperCase;
+
+    /**
+     * Construct a new instance which transforms to upper case.
+     */
+    public CaseNameRewriter() {
+        this.upperCase = true;
```

Comment:
The one-argument constructor can just be called here instead:

`this(true)`

Reply:
fixed!